pipelines/_ingredients/ApPipe.yaml

```diff
@@ -16,7 +16,8 @@ description: End to end Alert Production pipeline.
 # pipeline, e.g., $AP_PIPE_DIR/pipelines/$CAMERA/ApPipeCalibrateImage.yaml.
 
 imports:
-  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcd.yaml
+  - $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcd.yaml
+  - $ANALYSIS_TOOLS_DIR/pipelines/apDetectorVisitQualityCore.yaml
```

Comment:
Please add `analysis_tools` to `ups/ap_pipe.table`. Otherwise, setting up `ap_pipe` won't let you use its pipelines.